scripts/ci/check_pull_request.py

```diff
@@ -84,18 +84,24 @@ def check_pull_request(title, body):
         # only check title which start with `[`
         if title.startswith('['):
             error_flag = check_line(title)
-        is_edit_history_note = False
-        history_note_error_flag = False
-        for line in body:
-            if line.startswith('['):
-                # get component name in []
-                ref = re.findall(r'[\[](.*?)[\]]', line)
-                if ref and ref[0] not in ['Component Name 1', 'Component Name 2']:
-                    is_edit_history_note = True
-                    history_note_error_flag = check_line(line) or history_note_error_flag
-        # If the `History Notes` is edited:
-        # Use the history notes check result (history_note_error_flag), ignore the title check result (error_flag).
-        error_flag = error_flag if not is_edit_history_note else history_note_error_flag
+        if '**History Notes**' in body:
+            is_edit_history_note = False
+            history_note_error_flag = False
+            for line in body:
+                if line.startswith('['):
+                    # get component name in []
+                    ref = re.findall(r'[\[](.*?)[\]]', line)
+                    # exclude ['-c', 'a b', 'a b']
+                    # exclude ["-c"]
+                    char_not_allowed = ["'", '"', ',']
```

Comment:
It's actually hard to detect whether `[]` is an array or history tag. `[1]` will slip through this check.

Reply:
Add a func get_list to exclude all list types.